File: 백준/Bronze/15351. 인생 점수/인생 점수.py
T = int(input())
for _ in range(T):
    Q = input()
    dap = 0
    for q in Q:
        if q != ' ':
            dap += (ord(q)-64)
    if dap == 100:
        print('PERFECT LIFE')
    else:
        print(dap)

File: 백준/Bronze/15351. 인생 점수/인생 점수.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

function forLogic(T){
  if (T<=0) {
    process.exit(0);
    return;
  }

  rl.question('', (Q) => {
    let dap = 0;
    for (let i = 0; i < Q.length; i++){
      if (Q[i] !== ' '){
        dap += Q.charCodeAt(i) - 64;
      }
    }
    if (dap === 100){
      console.log('PERFECT LIFE')
    } else {
      console.log(dap);
    }

    forLogic(T-1);
  });
}

rl.question('', (T) => {
  const TC = parseInt(T);
  forLogic(TC);
});